slsReceiverSoftware/src/MasterFileUtility.cpp

```diff
@@ -215,7 +215,7 @@ std::string CreateVirtualHDF5File(
 
         // property list
         H5::DSetCreatPropList plist;
-        int fill_value = -1;
+        long fill_value = -1;
```

Comment:
Could you please change this to a uint64_t? Could you also change it in HDF5DataFile.cpp: there is also a variable 'fill_value' that is of datatype 'int' and that should be changed to 'uint64_t'?

Reply:
Yes, uint64_t is better. As to HDF5DataFile.cpp, I see the size of dataType in the next line (i.e. line 190) is at most 32bit, thus didn't do the change before, but using uint64_t is safer anyway. I have changed the type of fill_value to uint64_t in both files.